pyaedt/hfss.py

```diff
@@ -2447,8 +2460,17 @@ def create_scattering(self, plot_name="S Parameter Plot Nominal", sweep_name=Non
         -------
         bool
             ``True`` when successful, ``False`` when failed.
-            
+
+        Examples
+        --------
+
+        Create a scattering named ``"S Parameter Plot Nominal"``.
+
+        >>> hfss.create_scattering()
```

Comment:
Here the example is very short but if we decide to assign every parameter a value different from the default ones, we might have to create a new sweep, new ports... This could lead to a confusing example for the customer.

Reply:
Added a note to explain that we're just using the default parameters.

Reply:
Ok. I will add this information in the example for the next ones.